### PR TITLE
zh-translation: /docs/setup/install/helm/index.md

### PR DESCRIPTION
**Pull request by idefav**
ref: https://github.com/servicemesher/istio-official-translation/issues/1281

### CODE REVIEW

**Comment by idefav** [comment]
@googlebot  I signed it!

File: content/zh/docs/setup/install/helm/index.md [hunk 1]
```diff
@@ -20,86 +20,76 @@ window.onload = function(){
...
 {{< /warning >}}
 
-Follow this guide to install and configure an Istio mesh for in-depth evaluation or production use.
+请按照本指南安装和配置 Istio 网格，以进行深入评估或用于生产。
 
-This installation guide uses [Helm](https://github.com/helm/helm) charts that provide rich
-customization of the Istio control plane and of the sidecars for the Istio data plane.
-You can simply use `helm template` to generate the configuration and then install it
-using `kubectl apply`, or you can choose to use `helm install` and let
+这种安装方式使用 [Helm](https://github.com/helm/helm) chart 自定义 Istio 控制平面和 Istio 数据平面的 sidecar 。
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/helm/index.md]: 句尾的标点符号前不用加空格

Comment (idefav) [reply content/zh/docs/setup/install/helm/index.md]: updated

**Pull request merged**